### DISCUSSION TITLE
Classic themes: tutorial on moving away from widgets to template parts

### DISCUSSION THREAD

**Discussion by annezazu** [WordPress/developer-blog-content#254] [Topic Ideas]
Pulling out of [this GitHub issue](https://github.com/WordPress/gutenberg/issues/26170#top) and thinking about the future of patterns & template parts, [including exposing patterns for classic themes in 6.6](https://github.com/WordPress/gutenberg/issues/52150), I think we need a blog post talking about how classic theme authors [should move to include template parts](https://learn.wordpress.org/tutorial/using-block-template-parts-in-classic-themes/) instead of adding entity based blocks to widget sidebars. 

I'd see this as a part tutorial and part announcement/encouragement. I think a lot of material could be reused and it could include information about the future of exposing patterns and why that's a more viable pathway with classic themes, including for template parts.

**Comment by bph** [comment]
This topic was approved 5/2/24 and conversation continues on the issue
- #259
